bin/ssh_scan

```diff
@@ -11,14 +11,14 @@ require 'logger'
 
 #Default options
 options = {
-  :sockets => [],
-  :policy => File.expand_path("../../policies/mozilla_modern.yml", __FILE__),
-  :unit_test => false,
-  :timeout => 2,
-  :threads => 5,
-  :verbosity => nil,
-  :logger => Logger.new(STDERR),
-  :fingerprint_database => "./fingerprints.db"
+  "sockets" => [],
+  "policy" => File.expand_path("../../policies/mozilla_modern.yml", __FILE__),
+  "unit_test" => false,
+  "timeout" => 2,
+  "threads" => 5,
+  "verbosity" => nil,
+  "logger" => Logger.new(STDERR),
+  "fingerprint_database" => "./fingerprints.db"
```

Comment:
Switched these from symbols to string refs to make it easier from the JSON <=> Hash changes introduced with Worker API abstraction.  Let data mangling is better.